internal/controller/compute/server/server.go

```diff
@@ -55,7 +55,8 @@ func Setup(mgr ctrl.Manager, opts *utils.ConfigurationOptions) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(name).
 		WithOptions(controller.Options{
```

Comment:
I this opts.CtrlOpts.ForControllerRuntime() can be used too. But this works as well.

Reply:
Nice find I didn't see that method, I think it would be wise to use that instead so that if anything new is added in future they get picked up by default. Did you want me to edit all the other resources as well or leave it just as server for now?